File: constructor.php
<?php
//constructor
class constructor{
	public $name;
    public $age;
function __construct($n,$a){
	echo"from constructor block <br>";
	$this->name=$n;
	$this->age=$a;
}
function info()
{
	echo"Name :".$this->name."<br> Age :".$this->age;
}
}
$obj=new constructor("chetan",20);
$obj->info();

?>

File: Constructor.py
class Construct:
    def __init__(self,a,g):   #parameterized constructor
       self.age=a
       self.gender=g
    def show(self):
        print('Age :',self.age)
        print('Gender :',self.gender)

print("Show Details")

c1=Construct(18,'Female')
c1.show()

c=Construct(21,'Male')
c.show()
